src/api/entities/common/namespaces/Authorizations.ts

```diff
@@ -120,4 +124,35 @@ export class Authorizations<Parent extends Signer> extends Namespace<Parent> {
         return new AuthorizationRequest(args, context);
       });
   }
+
+  /**
+   * @hidden
+   * Needed to handle snake case from rpc request
```

Comment:
This took me a long time to figure out what was happening. I was in a circle of fixing things while breaking other things for longer than I'd like to admit.

This works, but really we should figure out why RPC stuff didn't get updated. I put the word `basic` in for copy pasta for snake case stuff.

I am hoping if we figure out the generate script stuff it will eliminate the need for these (more in conversion).

Reply:
The schema does play a role, but the actual types used are coming from `augment-api-rpc` and `augment-api-query`.

I think its weird the RPC one looks at `polymesh-types/polymesh` for its imports, the query one gets them from `@polkadot/types/lookup`.

There might be a reason for the divergence, but I'd think we can transform the rpc file like you are suggesting.

Reply:
In that file I changed `Authorization` to `PolymeshPrimitivesAuthorization` by hand and that seemed to work.

I think changing that file is the right approach. I am going to give that a shot